src/main/resources/config.yml

```diff
@@ -108,240 +110,263 @@ whole-number-prices-only: false
+  #Should QS send stock messages to staffs?
   sending-stock-message-to-staffs: false
-  #Should we disable trading for creative mode?
+  #Should QS disable trading in creative mode?
   disable-creative-mode-trading: false
-  #Should we disable super tool?(It's a tool which allow you break shop in creative mode by gold axe)
+  #Should QS disable the super tool?
+  #The Super Tool is a Golden Axe that allows creative mode players to break shops.
   disable-super-tool: false
-  #Should we allow the owner of shop break the sign (for sign replacement, but may cause sign dupe when auto-sign is true)?
+  #Should QS allow the owner of a shop to break the sign to let players replace the sign?
+  #This may cause a sign dupe when auto-sign is true!
   allow-owner-break-shop-sign: false
 
-  #Is there a fee for changing prices on a shop (Help deter endless undercutting)
+  #Should there be a fee for changing shop prices?
+  #This can help with endless price undercutting.
   price-change-requires-fee: true
-  #If price changes require a fee, how much is the fee
+  #The amount of this fee.
   fee-for-price-change: 50
 
-  #Should we try and lock their shops from other players, so people can't steal from them?
+  #Should QS try to lock shops for other players?
+  #This prevents players from stealing from them.
   lock: true
 
-  #Should we disable the quick way of creating shop? (click a chest with item)
+  #Should QS disable the quick creation of shop?
+  #To quickly create a shop click a chest with an Item.
   disable-quick-create: false
 
-  #Interact options
+  #Interaction settings.
   interact:
-    #Interact mode
-    #0=Only mode, Only can interact with sneak or normal interact
-    #1=Both mode, interact with sneak (if enabled) and normal interact will do
-    #2=Reversed mode, disable interact with sneak if you enable blow option
+    #Interaction mode
+    #0=Only interaction with sneak OR normal.
+    #1=Interaction with sneak (if enabled) AND normal.
+    #2=Disables interaction with sneak.
     interact-mode: 0
     sneak-to-create: false
     sneak-to-trade: false
     sneak-to-control: false
-    #Swap left and right click behavior
+    #Swap left and right click behavior.
     swap-click-behavior: false
 
   #Should we automatically create the sign for the chest?
   auto-sign: true
 
-  #Should we pay/take money to/from unlimited shops owner?
+  #Should we pay/take money to/from unlimited shops owners?
   pay-unlimited-shop-owners: false
 
-  #Should we place display items on the chests? This may cause duped items, especially for modded servers.
+  #Should we place display items on the chests? 
+  #This may cause duped items, especially for modded servers!
   display-items: true
 
-  #The plugin will check if the display item is in a valid position every specified amount of ticks. 0 to disable.
+  #QuickShop will check if the display item is in a valid position every specified amount of ticks. 
+  #Set to 0 to disable.
```

Comment:
Set to 0 to disable it?

Reply:
"0 to disable." -> "Set to 0 to disable."